app/views/layouts/application.html.erb

```diff
@@ -9,16 +9,13 @@
   <% if @canonical %>
     <link rel="canonical" href="<%= @canonical %>" />
   <% end %>
+  <%=raw @ranguba_template.head %>
 </head>
 <body>
-  <div class="header">
-    <%=raw @ranguba_template.header %>
-  </div>
+  <%=raw @ranguba_template.header %>
```

Comment:
Why is `div` removed?

Reply:
Because the example hader.txt and footer.txt also contain same class div. Anyway this change is not related to the topic, thus I separate it from this PR.

Reply:
Ah. We should remove it in samples.